src/main/resources/userEmailTemplate-feedbackSession.html

```diff
@@ -62,14 +62,9 @@
   </ul>
 </p>
 
-<p>
-  For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: ${coOwnersEmails}.
-</p>
+<p>No action is required if you have already submitted.</p>
```

Comment:
This one should not be here, otherwise there really is no difference between this and the `sessionClosing` template.

Let's examine the three cases where this template is used (based on your claim):
- `sessionClosed` -> nothing can really be done; the submission links will not work any more.
- `sessionOpening` -> it goes without saying that there is something to be done.
- `sessionReminder` -> here "No action is required" is appropriate. Why not make the `sessionReminder` one follow the `sessionClosing` template?

Reply:
Yes, we can do that. I will use `sessionClosing` template to generate `sessionReminder` emails too.